demo/26px/index.html

```diff
@@ -339,7 +339,7 @@ <h2 class="u-fs-lg u-mb">Wordmark Layout</h2>
           </svg>
         </span
         ><span class="l-grid__item u-1/2 u-mb u-truncate">
-          <svg class="relationshape u-fg-talk-yellow u-va-bottom">
+          <svg class="relationshape u-fg-talk-yellow">
```

Comment:
@ssidorchik note that special-case vertical alignment is no longer necessary with the updated `sunshine` icon in this PR.

Reply:
👍 great. Does it make sense to create separate color variable for Sunshine? I know it's the same as Talk, but because it's not clear enough why, e.g. final value is not available yet, or is really identical to Talk, so we left a [comment in Billing for clarification](https://github.com/zendesk/billing/blob/master/client/src/common/components/ProductBrands/SunshineBrand.js#L16)

Reply:
At this point, @allisonacs has directed folks to use the `talk` color for sunshine. If that changes and/or we think it makes sense to add an alias, that's fine. But it would be outside the context of this PR – rather added under `@zendeskgarden/css-variables`.